src/menu.scss

```diff
@@ -33,14 +33,13 @@ $block: #{$fd-namespace}-menu;
   // BLOCK BASE *******************************************
   // set all BLOCK reset and baseline styles
   @include fd-reset();
+  @include fd-ellipsis();
 
   position: relative;
-  white-space: nowrap;
   padding-top: fd-space(base);
   padding-bottom: fd-space(base);
   width: 100%;
-  display: table;
-  table-layout: fixed;
```

Comment:
table-layout: fixed assumes you have pre-computed all rendered scenarios. It it almost *never* a solution that should be used in production.